src/lib/features/personal-dashboard/personal-dashboard-read-model-type.ts

```diff
@@ -1,20 +1,24 @@
 import type { ProjectOwners } from '../project/project-owners-read-model.type';
 
 export type PersonalFeature = { name: string; type: string; project: string };
-export type PersonalProject = {
+export type BasePersonalProject = {
     name: string;
     id: string;
-    roles: {
+    roles?: {
         name: string;
         id: number;
         type: 'custom' | 'project' | 'root' | 'custom-root';
     }[];
 };
-export type PersonalProjectWithOwners = PersonalProject & {
-    owners: ProjectOwners;
+export type PersonalProject = BasePersonalProject & {
+    owners?: ProjectOwners;
```

Comment:
shouldn't roles also be moved in here? And why are they optional? Because we're not returning them for now? Will we later, though? Isn't that a separate endpoint?

Reply:
We're moving roles to a separate endpoint so will address the removal and adding to a new endpoint in a separate PR